.idea/.gitignore

```diff
@@ -1,2 +1,4 @@
 # Default ignored files
 /workspace.xml
+# Datasource local storage ignored files
+/dataSources/
```

Comment:
What do you think of ignoring the complete .idea folder, as it is related to the specific IDE you are using and not everyone will be using it (plus people may have different settings)

Reply:
@Zelig880 I'm so sorry. Totally slipped my mind.